helm/operator/values.yaml

```diff
@@ -279,6 +285,12 @@ console:
   containerSecurityContext:
     runAsUser: 1000
     runAsNonRoot: true
+    allowPrivilegeEscalation: false
```

Comment:
All other container security contexts also use `runAsGroup: 1000`, but the console doesn't. Should we add that too?

Reply:
I think that would be good, yes please @ramondeklein

Reply:
Fixed with 1611fa2735aa6ebdd50fce3f9a75e30ffde5db98.